person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age = nil, parent_permission: true, name: 'Unknown')
+    @id = Random.rand(1..10_000)
+    @age = age
+    @parent_permission = parent_permission
+    @name = name
+  end
+
+  def of_age?
+    @age >= 18
+  end
```

Comment:
- [ Required ] I think it would be better if you make this method `private` as stated in the project requirements, this will make it only accessible in the `Person` class as envisaged in the project requirements. I would like to suggest that you refactor this part of your code as shown below

## Example code
```ruby
class Person
  attr_accessor :name, :age
  attr_reader :id

  def initialize(age = nil, parent_permission: true, name: 'Unknown')
    @id = Random.rand(1..10_000)
    @age = age
    @parent_permission = parent_permission
    @name = name
  end

  def can_use_service?
    of_age? || @parent_permission
  end

  private
  def of_age?
    @age >= 18
  end
end
```

## Project requirements
![Screenshot from 2023-07-24 23-29-55](https://github.com/WilhelmK109/OOP-school-library/assets/16392046/08df93c0-c7d2-431f-a4b5-a9cc851be076)